requirements.txt

```diff
@@ -0,0 +1,19 @@
+#this file provides an alternative to set up via install.sh
```

Comment:
Might be helpful to be a bit more clear about what you mean by `alternative to set up via install.sh`. Do you mean that they are fully equivalent alternatives?

My current understanding is that `requirements.txt` does exactly the same thing as `install.sh` (except maybe installing git lfs)? If so, can't we just delete `install.sh` and use `requirements.txt` everywhere instead?